### ISSUE TITLE
tea section organization

### ISSUE THREAD

**Issue by rimmesbe** [goochua/TeaCozy_CodeAcad#2]
Don't put other elements on the same line as div tags. Div tags are structural and have block display, it makes it much hard to understand the layout when inline tags are on the same line.

Also, make sure you are using proper indentation here so it is easier to see the different components.

https://github.com/goochua/TeaCozy_CodeAcad/blob/master/index.html#L39-L53

**Comment by goochua** [comment]
Thank you for the feedback! In this instance are you referring to the <h4> content?

**Comment by goochua** [comment]
Thank you for the feedback! In this instance are you referring to the content on the h4 lines being included within the div?

**Comment by rimmesbe** [comment]
No, I mean the image tag being on the same line as the div tag.

https://github.com/goochua/TeaCozy_CodeAcad/blob/master/index.html#L39

**Comment by goochua** [comment]
Ah! Thank you again.
